internal/pkg/application/subscriptions/notifier.go

```diff
@@ -84,15 +86,38 @@ func (n *notifier) EntityCreated(ctx context.Context, e types.Entity) {
 		n.queue <- func() {
 			defer func() { tracing.RecordAnyErrorAndEndSpan(err, span) }()
 
-			err = postNotification(ctx, e, n.endpoint)
-			if err != nil {
-				logger.Error().Err(err).Msg("failed to post notification")
+			for _, t := range n.cfg.Tenants {
+				if strings.EqualFold(t.ID, tenant) {
+					for _, not := range t.Notifications {
+						for _, ent := range not.Entities {
+							if ent.Type != e.Type() {
+								continue
+							}
+
+							regexpForID, err := regexp.CompilePOSIX(ent.IDPattern)
+							if err != nil {
+								continue
+							}
+
+							if !regexpForID.MatchString(e.ID()) {
+								continue
+							}
+
+							go func(endpoint string) {
+								err = postNotification(ctx, e, endpoint)
```

Comment:
Här finns kanske en risk att kontexten som skickas in i EntityCreated canceleras innan vi hinner utföra postNotification. I så fall blir det en panic tror jag.